ios/sdk/src/picture-in-picture/DragGestureController.swift

```diff
@@ -17,14 +17,14 @@
 import UIKit
 
 final class DragGestureController {
-
     var insets: UIEdgeInsets = UIEdgeInsets.zero
+    var currentPosition: CustomPiPViewCoordinator.Position? = nil
```

Comment:
`CustomPiPViewCoordinator` it's not available. You need to rename to `PiPViewCoordinator` to compile the code.